src/test/java/com/plivo/api/AddressTest.java

```diff
@@ -0,0 +1,183 @@
+package com.plivo.api;
+
+import static com.plivo.api.TestUtil.loadFixture;
+import static junit.framework.TestCase.assertEquals;
+
+import com.plivo.api.models.address.Address;
+import com.squareup.okhttp.mockwebserver.MockResponse;
+import com.squareup.okhttp.mockwebserver.RecordedRequest;
+import org.junit.Test;
+
+public class AddressTest extends BaseTest {
+  @Test
+  public void addressCreateShouldSucceed() throws Exception {
+    expectResponse("addressCreateResponse.json", 201);
+
+    Address.creator("FR",
+      "Mr",
+      "Bruce",
+      "Wayne",
+      "128",
+      "RUE DU COMMANDANT GUILBAUD",
+      "PARIS",
+      "PARIS",
+      "75016")
+      .create();
+
+    assertRequest("POST", "Verification/Address/");
```

Comment:
Assert Request Body also

` assertEquals(body.get("callback_url"), "http://example.answer.url/");
    assertEquals(body.get("callback_method"), null);`

Reply:
As discussed this will be clubbed with phlo changes. Part of the code for the same are already checked in phlo branch